### ISSUE TITLE
Evaluation on CMU Panoptic

### ISSUE THREAD

**Issue by HuiliCui** [Arthur151/ROMP#167]
Hi, I read your code for Panoptic evaluation and want to make sure that the mesh parameter map information is fetched directly from the GT and not from the center map information , right?
code in romp/lib/maps_utils/result_parser.py line 97-100

**Comment by Arthur151** [comment]
No, we use the center map for sampling, sorry about that bug. Please set `match_preds_to_gts_for_supervision: True` during evaluation to make use of predicted center.
